src/content/nist.gov/SP800-53/rev4/xml/NIST_SP-800-53_rev4_HIGH-baseline_profile.xml

```diff
@@ -1,27 +1,12 @@
 <!-- Produced by SP800-53-profile-with-filter.xsl 2018-05-14-04:00
         runtime parameter settings: $baseline='HIGH'
      Adjusted by hand 2019-05-28. -->
-<profile xmlns="http://csrc.nist.gov/ns/oscal/1.0"
-   id="uuid-56285eb5-54a4-4b56-ba03-45d1b78e9dfc">
+<profile xmlns="http://csrc.nist.gov/ns/oscal/1.0" id="uuid-56285eb5-54a4-4b56-ba03-45d1b78e9dfc">
    <metadata>
       <title>NIST Special Publication 800-53 Revision 4 HIGH IMPACT BASELINE</title>
       <publication-date>September 2018</publication-date>
```

Comment:
We need a more explicit format for dates. Something like what xml date uses.

Reply:
Can you create an issue for this?

Reply:
See #399.